packages/orchestrator-ui-components/src/components/WFOTable/utils/withSortableAndFilterableValues.ts

```diff
@@ -0,0 +1,23 @@
+import { WFOTableColumns, WFOTableDataColumnConfig } from './columns';
+
+export function withSortableAndFilterableValues<T>(
+    tableColumnConfig: WFOTableColumns<T>,
+    sortableFieldNames: string[],
```

Comment:
Can this be keyof T?

Reply:
I do want that too, BUT the backend is typing this as string as well. Therefore we (frontend) cannot guarantee what keys will appear here.

Reply:
We have a ticket to remove the strict typing : https://github.com/workfloworchestrator/orchestrator-ui/issues/290
Will refer to this discussion, to agree in that ticket what we eventually want to do in the project